src/starknet-wrappers.ts

```diff
@@ -276,11 +285,39 @@ export abstract class StarknetWrapper {
             prepared.push("--account_dir", options.accountDir);
         }
 
+        prepared.push("--chain_id", "0x534e5f474f45524c49");
```

Comment:
Instead of hardcoding this, it should be taken from the options object, same as in `prepareInteractOptions`. But this doesn't mean the user should specify it, but that whoever is calling this method (not the user), needs to provide it (see places which ultimately lead to `prepareInteractOptions` being called)

Reply:
Great catch!! https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/246/commits/368bcfb2129f10a4091654e14a60550131d5117e